pandas/core/frame.py

```diff
@@ -4479,6 +4479,14 @@ def quantile(self, q=0.5, axis=0, numeric_only=True):
             0 <= q <= 1, the quantile(s) to compute
         axis : {0, 1}
             0 for row-wise, 1 for column-wise
+        interpolation : {'linear', 'lower', 'higher', 'midpoint', 'nearest'}
+            Specifies the interpolation method to use, when the desired quantile lies between two data points i and j:
```

Comment:
BTW, you can just copy it from the source from the numpy.percentile function (using `np.percentile??` in IPython):

```
interpolation : {'linear', 'lower', 'higher', 'midpoint', 'nearest'}
    This optional parameter specifies the interpolation method to use,
    when the desired quantile lies between two data points `i` and `j`:
        * linear: `i + (j - i) * fraction`, where `fraction` is the
          fractional part of the index surrounded by `i` and `j`.
        * lower: `i`.
        * higher: `j`.
        * nearest: `i` or `j` whichever is nearest.
        * midpoint: (`i` + `j`) / 2.

    .. versionadded:: 1.9.0
```